Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/FastDepthAwareBlurCommon.azsli

```diff
@@ -109,7 +109,7 @@ ShaderResourceGroup PassSrg : SRG_PerPass
         // The higher this value, the sharper edges will appear
         float m_depthFalloffStrength;
 
-        uint3 PADDING;
+        uint PADDING;
```

Comment:
It seems adding 4 bytes padding will sum to 40 bytes. Is that correct?
Also, we may need to rename it to m_padding to match the other padding variables.

Reply:
I've compared this with uint3 and uint and this is indeed the case.